api_tests/crossref/views/test_crossref_email_response.py

```diff
@@ -158,7 +158,7 @@ def test_wrong_request_context_raises_permission_error(self, app, url, error_xml
     def test_error_response_sends_message_does_not_set_doi(self, app, url, preprint, error_xml):
         assert not preprint.get_identifier_value('doi')
 
-        with mock.patch('framework.auth.views.mails.send_mail') as mock_send_mail:
+        with mock.patch('framework.auth.views.mails.execute_email_send') as mock_send_mail:
```

Comment:
Can you make this a more reusealbe fixture style mock like the ones in the conftest.py such as mock_share_responses or mock_akismet? Just call it mock_email make sure that allows mocks `USE_EMAIL` `USE_CELERY` and any other config settings, include all services, send_grid and general SMTP.